.development/pyboard_util.py

```diff
@@ -187,4 +187,32 @@ def end_copy_via_repl(args):
     pass
 
 def clean_via_repl(args):
-    raise Exception("not implemented yet")
+    init_copy_via_repl(args)
+    print("Cleaning:", end=" ", flush=True)
+    try:
+        execbuffer(get_pyb(args), "clean()")
+    except PyboardError as er:
+        print("FAIL")
+        print(er)
+        pyb.close()
```

Comment:
is `pyb` defined here? The equivalent `hard_reset()` function reads it before the try/except block.

Reply:
sorry, this ended up in the wrong branch - will fix :(

Reply:
pushed all commits to the correct branch now, sorry for the confusion.